lms/static/sass/inline-discussion.scss

```diff
@@ -0,0 +1,22 @@
+// From lms-main-v1.scss
```

Comment:
@andy-armstrong This commit addresses your comment: https://github.com/edx/edx-platform/pull/13483/commits/524599ea2e469fc90a14ec71ac29c4d7a99ad328.